automation/MPLogger.py

```diff
@@ -67,14 +69,85 @@ def loggingserver(log_file, status_queue):
     A logging server to serialize writes to the log file from multiple
     processes.
 
-    <log_file> location of the log file on disk
-    <status_queue> is a queue connect to the TaskManager used for communication
+    Parameters
+    ----------
+    log_file : string
+        The location of the log file on disk
+    status_queue : Queue
+        A queue to connect to the TaskManager process
     """
+
+    # If running a cloud crawl, we can pull the sentry endpoint and related
+    # config varibles from the environment
+    SENTRY_DSN = os.getenv('SENTRY_DSN', None)
+    if SENTRY_DSN:
+        sentry_logging = LoggingIntegration(
+            level=logging.DEBUG,
+            event_level=logging.ERROR
+        )
+        sentry_sdk.init(
```

Comment:
I am not sure that having multiple sentry_sdk.init is a good idea. Not sure what the alternative is but init is already in https://github.com/mozilla/OpenWPM/blob/a337b52eb172ceed9e2ab092a4ba0e1f7421ad29/crawler.py#L30

Reply:
I believe this is required since `crawler.py` and `MPLogger.py` execute in different processes.

Reply:
Removing the crawler.py code sounds like a good idea to me

Reply:
For some background, MP logger was written to serialize writes from multiple processes (the task manager, each browser manager, and each extension). See https://mattgathu.github.io/multiprocessing-logging-in-python/ for a description of the problem and an alternate solution (using queues instead of sockets). So by initializing and attaching sentry to MPLogger's logging module, we should capture logs from all of those processes -- basically everything that would normally be written to the log file on disk. However, any uncaught exceptions in the main process (i.e., the process that runs the `crawler.py` crawl script and the TaskManager) will not be captured by MPLogger.

Reply:
Thus, by initializing in both we should capture exceptions in both.

Reply:
Ok great to know. Initializing in both seems fine, but we should move the init code into a separate file and avoid dry.

Reply:
Makes sense! I also think it might make sense to just run the logging server within a thread of the main process -- it doesn't really require a new process. Then only one initialization should be fine (if I understand correctly how the logger is scoped).

Reply:
Feel free to move that into the same process in a separate PR - or we could consider using fluentd or similar which would allow us to configure persistent logging for containerized deployments - but ultimately out of scope for this particular PR. :)